packages/dtslint/src/lint.ts

```diff
@@ -116,13 +116,8 @@ function getEslintOptions(
       overrides: [
         {
           files: allFiles,
-          rules: skipNpmNaming
-            ? {}
-            : {
-                "@definitelytyped/npm-naming": "error",
-              },
-        },
-      ],
+        }
+      ], 
```

Comment:
I made what looked like the minimal correct change here because I don’t understand the config scheme, but it seems likely that this can be reduced more somehow.

Reply:
Yes, it at this point you could just drop baseConfig.